run.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
-python scripts/check_requirements.py requirements.txt
-if [ $? -eq 1 ]
-then
+if [ ! -d "./env_check.lock" ];then
+    python scripts/check_requirements.py requirements.txt
+    if [ $? -eq 1 ]
+    then
     echo Installing missing packages...
     pip install -r requirements.txt
+    fi
+    touch env_check.lock
 fi
 python -m autogpt $@
```

Comment:
Replaced the variable $@ with "$@" to properly handle arguments containing spaces.
```suggestion
python -m autogpt "$@"
```